src/pages/ShikshalokamVoiceChat/dynamic-voice-chat.js

```
@@ -0,0 +1,2757 @@
+
+    return <PdfDownloader key={new Date().getTime()} storyData={storyData} isShikshalokam={true} downloadTriggered={triggerDownload} handleDownloadStop={handleDownloadStop} storyMediaArr={files} currentState={currentState} current_company={current_company} />
+  }
+
+  const handleSelectedTypeNameChanges = e => {
+    function changeSelectedValue(value, e) {
+      if (value === "") value = selectedLabel?.types[0]?.value
+      setSelectedType(value)
+      resetChat(e)
+    }
+
+    let { value } = e?.target
```

Biome (2.3.14)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)